iphone/Maps/LocalizedStrings/pt.lproj/Localizable.stringsdict

```diff
@@ -49,11 +44,11 @@
 			<string>NSStringPluralRuleType</string>
 			<key>NSStringFormatValueTypeKey</key>
 			<string>d</string>
-			<key>one</key>
-			<string>%d trajeto</string>
+			<key>zero</key>
+			<string>%d trilhos</string>
```

Comment:
Might be worth spot checking this zero/one change to make sure it's intended and correct